### DISCUSSION TITLE
Can’t signup for cloud platform

### DISCUSSION THREAD

**Discussion by uppercaseVar** [Budibase/budibase#3734] [Help]
I have been trying to sign up for the cloud offering of Budibase, but it seems when I try to sign up for the cloud platform I land in an endless verify email loop; where I cant sign in without verifying my email and I cant verify my email until I have signed in. Any help is appreciated!

**Comment by mjashanks** [comment]
Hi there @User-042 - we have had this reported before, but didn't get a response from them, so weren't able to investigate!

So, it would be good if you could give us some additional information:

- What browser are you using?
- Are you using any browser extensions?
- Are you able to let us know what email you signed up with. You can send this to support@budibase.com, to keep it private.

**Reply by uppercaseVar** [reply]
I have sent an email to [support@budibase.com](mailto:support@budibase.com).

**Reply by Rory-Powell** [reply]
Thanks for the details @User-042. Can you clarify what you are experiencing when you click the link in your email inbox, is this bringing you straight to login instead of the verification screen? To try to debug further could you copy the link sent in the verification email and paste it into a new private window?

**Comment by uppercaseVar** [comment]
I don’t have the verification email. It could have landed in spam and I
just forgot about it. I really have no clue.

**Reply by Rory-Powell** [reply]
I've re-sent your verification email @User-042 which should be active for 24 hours

**Comment by uppercaseVar** [comment]
I received it and clicked on it but I get the error message, “Error
creating tenant”. And when I login it still says “Account not verified”

**Reply by Rory-Powell** [reply]
Thanks @User-042 this was useful, your tenant (budibase domain) name starting with a number is the issue as it isn't a supported pattern for database names in couchdb. I've manually deprovisioned the account you created, you should be able to run through account creation again with a different name

**Reply by Rory-Powell** [reply]
If the registration flow always brings you back to the verification page, you should be able to clear down the local storage for account.budibase.app like so using the browser dev tools

![Screenshot 2021-12-15 at 10 18 37](https://user-images.githubusercontent.com/8755148/146168287-45ca17ce-a2e8-4502-98d1-f0c39a6d90a5.png)

alternatively using a private browser will work
